bazel/ui.bzl

```diff
@@ -116,6 +116,9 @@ def _pl_ui_test_impl(ctx):
         test_cmd = [
             "yarn coverage_ci",
             "sed -i \"s|SF:src|SF:src/ui/src|g\" coverage/lcov.info",
+            "sed -i \"s|^DA:[1-9],0||g\" coverage/lcov.info",  ## remove license headers that show up as uncovered
+            "sed -i \"s|^DA:1[0-8],0||g\" coverage/lcov.info",  ## remove license headers that show up as uncovered
```

Comment:
Probably should adjust the comment here to explain that the license header is a constant 17 lines long (plus newline after for 18), what the 'DA:' lines in `lcov.info` mean, and so on.

Reply:
no longer relevant. :)